README.md

```diff
@@ -46,4 +48,10 @@ terraform destroy -parallelism=1
 
 ## Troubleshooting
 
-see [steps](https://github.com/threefoldtech/terraform-provider-grid/blob/development/TROUBLESHOOTING.md)
\ No newline at end of file
+see [steps](https://github.com/threefoldtech/terraform-provider-grid/blob/development/TROUBLESHOOTING.md)
+
+## Run tests
+To run the tests, run the following command
+```bash
+go test ./...
```

Comment:
please add necessary environment variables (e.g. `PUBLICKEY=$(cat ~/.ssh/id_rsa.pub) TF_LOG=DEBUG`), and tests should run [sequentially](https://github.com/threefoldtech/terraform-provider-grid/issues/12), the mount test failed with log specifying this error.